### PR TITLE
prod.py : add specific jormungandr postgres / pgpool variables

### PR DESCRIPTION
**Pull request by fdegomme-canaltp**

### CODE REVIEW

File: templates/jormungandr/settings.py.jinja [hunk 1]
```diff
@@ -6,8 +6,14 @@
...
 INSTANCES_DIR='{{env.jormungandr_instances_dir}}'
 
-#chaine de connnection à postgresql pour la base jormungandr
-SQLALCHEMY_DATABASE_URI ='postgresql://{{env.tyr_postgresql_user}}:{{env.tyr_postgresql_password}}@{{env.tyr_postgresql_database_host}}/{{env.tyr_postgresql_database}}'
+#chaine de connnection à postgresql (pgpool) pour la base jormungandr
+{% if env.name == "prod" %}
+SQLALCHEMY_DATABASE_URI ='postgresql://{{env.jormungandr_postgresql_user}}:{{env.jormungandr_postgresql_password}}@{{env.jormungandr_postgresql_database_host}}:{{env.jormungandr_postgresql_database_port}}/{{env.jormungandr_postgresql_database}}'
+{% else %}
+# There is a dirty condition there to keep compatibility with staging servers to avoid changing all environments configuration about them in navitia_deployment_conf like prod.py
+# One day, we will must do this change
```
Comment (pbougue) [inline templates/jormungandr/settings.py.jinja]: will or must? ;)

File: templates/jormungandr/settings.py.jinja [hunk 2]
```diff
@@ -6,8 +6,14 @@
 #emplacement ou charger les fichier de configuration par instances
 INSTANCES_DIR='{{env.jormungandr_instances_dir}}'
 
-#chaine de connnection à postgresql pour la base jormungandr
-SQLALCHEMY_DATABASE_URI ='postgresql://{{env.tyr_postgresql_user}}:{{env.tyr_postgresql_password}}@{{env.tyr_postgresql_database_host}}/{{env.tyr_postgresql_database}}'
+#chaine de connnection à postgresql (pgpool) pour la base jormungandr
+{% if env.name == "prod" %}
```
Comment (pbougue) [inline templates/jormungandr/settings.py.jinja]: feels dirty, maybe test if it's in deployment conf file
but the best would be to use it as much as possible on other platforms to test it before prod (and it doesn't look too complicated, but not sure).

File: templates/jormungandr/settings.py.jinja [hunk 3]
```diff
@@ -6,8 +6,14 @@
...
+SQLALCHEMY_DATABASE_URI ='postgresql://{{env.jormungandr_postgresql_user}}:{{env.jormungandr_postgresql_password}}@{{env.jormungandr_postgresql_database_host}}:{{env.jormungandr_postgresql_database_port}}/{{env.jormungandr_postgresql_database}}'
+{% else %}
+# There is a dirty condition there to keep compatibility with staging servers to avoid changing all environments configuration about them in navitia_deployment_conf like prod.py
+# One day, we will must do this change
+SQLALCHEMY_DATABASE_URI='postgresql://{{env.tyr_postgresql_user}}:{{env.tyr_postgresql_password}}@{{env.tyr_postgresql_database_host}}/{{env.tyr_postgresql_database}}'
```
Comment (antoine-de) [inline templates/jormungandr/settings.py.jinja]: roh the change is not that complicated in the configuration (s/tyr_postgresql/jormungandr_postgresql/ ?)  , and would be much cleaner :stuck_out_tongue_winking_eye:

**Comment by Hoshiyo** [comment]
on merge mais avec une demande de refacto  derriere https://jira.canaltp.fr/browse/DEVOPP-361

**Pull request merged**